docs/feature_encoders.md

```diff
@@ -40,14 +40,27 @@ It can also be defined per-encoder, by instead defining:
 
 ## Split Keyboards
 
-If you are using different pinouts for the encoders on each half of a split keyboard, you can define the pinout (and optionally, resolutions) for the right half like this:
+The above is enough for split keyboards that are symmetrical, i.e. the halves have the same number of encoders and they are on the same pins.
+If the halves are not symmetrical, you can define the pinout (and optionally, resolutions) of the right half separately.
+The left half will use the definitions above.
 
 ```c
 #define ENCODERS_PAD_A_RIGHT { encoder1a, encoder2a }
 #define ENCODERS_PAD_B_RIGHT { encoder1b, encoder2b }
 #define ENCODER_RESOLUTIONS_RIGHT { 2, 4 }
 ```
 
+If only the right half has encoders, you must still define an empty array for the left pads (and resolutions, if you define `ENCODER_RESOLUTIONS_RIGHT`).
+
+```c
+#define ENCODERS_PAD_A {  }
+#define ENCODERS_PAD_B {  }
+#define ENCODER_RESOLUTIONS {  }
```

Comment:
It may be worth removing this.  

In the encoder.c file, you could add: 

```c
#ifndef ENCODERS_PAD_A
#    define ENCODERS_PAD_A {  }
#endif
```

Or the like.

Reply:
I was a bit worried about making things confusing. Leaving out the left side would have very different effect from leaving out the right side. Leave out the `_RIGHT` pads, the pads get mirrored and you get encoders on both sides. But leave out the "normal" (aka. left) pads and you only get encoders on right.

It would also mean that a board with encoders on only left would look different from a board with encoders only on right:

```c
// only right
#define ENCODERS_PAD_A_RIGHT { X }
#define ENCODERS_PAD_A_RIGHT { Y }
// only left
#define ENCODERS_PAD_A { X }
#define ENCODERS_PAD_A { Y }
#define ENCODERS_PAD_A_RIGHT {  }
#define ENCODERS_PAD_A_RIGHT {  }
```

Waddaya think?

Reply:
Personally I'd prefer to explicitly require both sides in definitions, if an asymmetrical encoder setup is defined.
So you _could_ do the `#ifdef`, but I'm not sure it's the most readable/understandable/maintainable.

So yeah, my vote would be both:
```c
// only right
#define ENCODERS_PAD_A { }
#define ENCODERS_PAD_A { }
#define ENCODERS_PAD_A_RIGHT { X }
#define ENCODERS_PAD_A_RIGHT { Y }
// only left
#define ENCODERS_PAD_A { X }
#define ENCODERS_PAD_A { Y }
#define ENCODERS_PAD_A_RIGHT { }
#define ENCODERS_PAD_A_RIGHT { }
```